### DISCUSSION TITLE
Missing auth docs

### DISCUSSION THREAD

**Discussion by howesteve** [supabase/supabase#3102] [Show and tell]
So in https://supabase.io/docs/guides/auth we have:

"When users sign up, Supabase assigns them a unique ID. You can reference this ID anywhere in your database. For example, you might create a profiles table referencing the user using a user_id field." - and no other info available.

It took me a good time to guess those infos are on a separate auth schema:

select * from auth.users;

Please, this should be documented.

**Comment by thebengeu** [comment]
Thanks Steve for your feedback! There are a few references to `auth.users` further down that doc, but this could indeed be clearer up top, sent out #3139 to fix.
